styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -18,15 +18,15 @@
     @extend .white;
     @extend .left;
   }
-  p{
+  p {
     letter-spacing: 1px;
     line-height: 24px;
     font-size: 16px;
     padding-bottom: 10px;
   }
 }
 
-#hosting-options{
+#hosting-options {
```

Comment:
I'm all about cleaning sass lint errors, but if it's unrelated to this PR, not sure it's worth the effort.  But if we do decide we're okay with small incremental fixes like this, then we need to make sure the `package.json` file gets updated to reference the new error count. 

This more seems like an trade-off to allow one new error into the system by getting rid of two.  No new code should introduce new code to sass lint. If it does we need to figure evaluate if we want to exclude, or modify the rules instead.